src/Uno.UI.RuntimeTests/Tests/Windows_Globalization/Given_DateTimeFormatter.cs

```diff
@@ -79,6 +79,8 @@ public void When_FormattingDateVariants_ShouldProduceExpectedFormats()
 	{
 		var expectedResults = new Dictionary<string, string>
 		{
+			{ "day", "27" },
```

Comment:
The issue with the DatePicker was not caught earlier because of this assertion. The Day's Part of the Picker uses this format.